themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml

```diff
@@ -47,6 +47,7 @@
   <?php if (!$this->facet['isApplied'] && null !== $this->facet['count']): ?>
     <span class="badge">
       <?=$this->localizedNumber($this->facet['count']) ?>
+      <span class="sr-only"><?=$this->transEsc('results') ?></span>
```

Comment:
In order to more fully take advantage of i18n, what if you did it like this?

```
<?php $currentCount = $this->localizedNumber($this->facet['count']); ?>
<span class="sr-only"><?=$this->transEsc('result_count', ['%%count%%' => $currentCount]) ?></span>
<span aria-hidden><?=$currentCount ?></span>
```

This way, the screen reader user gets a fully-articulated count appropriate to their local language and the stray number is hidden, and the visual user sees no change.

Reply:
very clever. feels like i'm wasting a lot of your time! Resolved it.

Reply:
Not a waste, it's a good collaboration! :-)